### DISCUSSION TITLE
How to do ductile fracture with phase-field method in MOOSE?

### DISCUSSION THREAD

**Discussion by rh201** [idaholab/moose#23981] [Q&A General] [answered]
Hello MOOSEer,

I'm writing to ask if there are any examples on ductile fracture with phase-field method in MOOSE or other MOOSE-based applications? Thanks.

**Comment by GiudGiud** [comment]
Hello

I think @tonkmr merged some models doing this. I dont know if there's a tutorial but there are test inputs
I ll let him confirm

Guillaume

**Reply by jiangwen84** [reply]
I would suggest you look at https://github.com/hugary1995/raccoon

**Reply by tonkmr** [reply, marked as answer]
@GiudGiud is referring to https://mooseframework.inl.gov/moose/source/materials/ComputeCrackedStress.html. There is an example there with a finite strain plastic material.

**Reply by rh201** [reply]
Thanks !
